src/components/FeatureViews/Search/SearchResults.js

```diff
@@ -275,7 +231,7 @@ const SearchResults = (props) => {
                                     </Grid>
                                 </Grid> : ""
                         }
-                        {account_num_results && course_num_results !== 0&&searchState.primaryFilter !== SEARCH_ACCOUNTS? <hr /> : ""}
+                        {accountResultsNum && courseResultsNum !== 0&&searchState.primaryFilter !== SEARCH_ACCOUNTS? <hr /> : ""}
```

Comment:
`accountResultsNum` as a boolean condition checks against all falsey values, including 0, so the second condition is unnecessary.